### PR TITLE
Scientific Calculator Completed With All Functionality

### PR DESCRIPTION
**Pull request by rutulsheladiya31**
Scientific Calculator Completed With All Functionality As per LMS.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+}
+
+
+
+// Memory Functions
+document.querySelectorAll('.memoryfunction').forEach((element) => {
+    element.addEventListener('click', () => {
+        memoryFunction(element)
+            // console.log(true)
+        console.log(`stored ${memData} data`)
```
Comment (kerulshahsimform) [inline js/script.js:207]: Console log is there. This is recurring feedback point.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                        <button class="btn-white" onclick="addData('.')">.</button>
+                        <button onclick="equalto()" style="background-color: rgb(132, 182, 249);">=</button>
+                    </td>
+                </tr>
+            </table>
+        </div>
+    </section>
+
+    <!-- Js -->
+    <script src="js/bootstrap.bundle.min.js"integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
```
Comment (kerulshahsimform) [inline index.html:135]: is there any use of integrity and cross origin attribute if they are called locally?

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,259 @@
+let inputData = document.getElementById("inputValue");
+let tempData;
+let power_Xy_Flag = false;
+let changeOperation_Flag = true;
```
[the remark this answers is not in the document]
Comment (rutulsheladiya31) [reply js/script.js]: Solved.

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+        changeOperation_Flag = true
+    }
+}
+
+function pi() {
+    inputData.value = inputData.value + Math.PI
+}
+
+function e() {
+    inputData.value = inputData.value + Math.E
```
[the remark this answers is not in the document]
Comment (rutulsheladiya31) [reply js/script.js:93]: Yeah Got it. i changed that function name.

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+        }
+        if (!memoryFlag) {
+            memData = "";
+            document.getElementById("mc").style.color = "Black";
+            document.getElementById("mr").style.color = "Black";
+            document.getElementById("ms").style.color = "Black";
+            memoryFlag = true;
+        }
+
+        if (memoryFlag) memData += eval(inputData.value) + "+";
```
[the remark this answers is not in the document]
Comment (rutulsheladiya31) [reply js/script.js]: Okay Resolved it.

File: js/script.js (hunk 1 above)
Comment (rutulsheladiya31) [reply js/script.js:206]: Resolved.

File: index.html (hunk 2 above)
Comment (rutulsheladiya31) [reply index.html]: No,if we are called file locally then there is no need to keep that.

File: js/script.js [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+}
+
+function absolute() {
+    let temp = eval(inputData.value).toString().substring(0, 1)
+    if (temp == '+' || temp == '-') {
+        inputData.value = eval(inputData.value).toString().substring(1)
+    }
+}
+// 7th row
+function square_root() {
```
Comment (kerulshahsimform) [inline js/script.js:132]: as per PSR4 standards, From next tasks, make sure there will be camelCase in function name and variable name both,.

File: js/script.js (hunk 1 above)
Comment (kerulshahsimform) [reply js/script.js:206]: This is still not resolved.

File: js/script.js [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+    element.addEventListener('click', () => {
+        memoryFunction(element)
+        console.log(`stored ${memData} data`)
+    })
+})
+
+function memoryFunction(ele) {
+    let eleId = ele.attributes.id.value;
+    if (eleId == "mp") {
+        console.log(memData)
```
Comment (kerulshahsimform) [inline js/script.js:213]: here is also console.log

Comment (rutulsheladiya31) [reply js/script.js]: Removed.

File: js/script.js (hunk 6 above)
Comment (rutulsheladiya31) [reply js/script.js:132]: Okay Sure

**Review by kerulshahsimform — APPROVED** [review]

**Pull request merged**